src/compile/axis/parse.ts

```diff
@@ -296,51 +296,51 @@ function getProperty<K extends keyof AxisComponentProps>(
 
   switch (property) {
     case 'scale':
-      return model.scaleName(channel);
+      return model.scaleName(channel) as any;
```

Comment:
Why so many `as any` here?  -- Can we do something like direction in the legend?